src/sonic-ctrmgrd/tests/docker.py

```diff
@@ -0,0 +1,7 @@
+#! /usr/bin/python3
```

Comment:
This file does not appear to be an executable script. Please remove shebang.

Reply:
This is the file loaded, while running unit tests.

Reply:
But it's not executable, so it doesn't need a shebang.

Reply:
Removed.